### PR TITLE
Update loader for new format

### PR DESCRIPTION
**Pull request by forrestfwilliams**

### CODE REVIEW

File: severe-weather/loader.py [hunk 1]
```diff
@@ -36,86 +37,65 @@ def __call__(self, batch):
...
+        self.chip_names = sorted(self.chip_names)
+        labels = list(self.label_path.glob('*.zarr.zip'))
+        rtcs = list(self.rtc_path.glob('*.zarr.zip'))
+        s2s = list(self.s2_path.glob('*.zarr.zip'))
+        self.chip_list = []
+        for name in self.chip_names:
+            label = [x for x in labels if name in x.name][0]
+            rtc = [x for x in rtcs if name in x.name][0]
+            s2 = [x for x in s2s if name in x.name][0]
+            self.chip_list.append({'LABEL': label, 'S1RTC': rtc, 'S2L2A': s2})
```
Comment (williamh890) [inline severe-weather/loader.py]: How do you feel about remove member variables that are not used outside of the constructor?
```suggestion
        label_path = chip_path / split / 'LABEL'
        rtc_path = chip_path / split / 'S1RTC'
        s2_path = chip_path / split / 'S2L2A'
        chip_names = ['_'.join(x.name.split('_')[-4:]).split('.')[0] for x in label_path.glob('*.zarr.zip')]
        chip_names = sorted(chip_names)
        labels = list(label_path.glob('*.zarr.zip'))
        rtcs = list(rtc_path.glob('*.zarr.zip'))
        s2s = list(s2_path.glob('*.zarr.zip'))
        self.chip_list = []
        for name in chip_names:
            label = [x for x in labels if name in x.name][0]
            rtc = [x for x in rtcs if name in x.name][0]
            s2 = [x for x in s2s if name in x.name][0]
            self.chip_list.append({'LABEL': label, 'S1RTC': rtc, 'S2L2A': s2})
```

Comment (forrestfwilliams) [reply severe-weather/loader.py]: Good idea!

File: severe-weather/loader.py [hunk 2]
```diff
@@ -202,25 +174,14 @@ class SatChipDataModule(NonGeoDataModule):
...
-        }
-
-        self.training_transforms = A.Compose([
-            A.D4(),
-            ToTensorV2()
-        ])
+        means = {'S1RTC': torch.Tensor(self.s1rtc_mean), 'S2L2A': torch.Tensor(self.s2l2a_mean)}
+        stds = {'S1RTC': torch.Tensor(self.s1rtc_std), 'S2L2A': torch.Tensor(self.s2l2a_std)}
+        self.training_transforms = A.Compose([A.CenterCrop(width=256, height=256), A.D4(), ToTensorV2()])
+        self.training_transforms = A.Compose([A.D4(), ToTensorV2()])
```
Comment (williamh890) [inline severe-weather/loader.py]:
```suggestion
        self.training_transforms = A.Compose([A.CenterCrop(width=256, height=256), A.D4(), ToTensorV2()])
```

File: severe-weather/loader.py [hunk 3]
```diff
@@ -36,86 +37,64 @@ def __call__(self, batch):
...
-        return len(self.label_ds.sample)
+        return len(self.chip_list)
 
     def __getitem__(self, index: int) -> dict[str, Any]:
-        sample_data = self.label_ds.isel(sample=index, x=self.slice_range, y=self.slice_range).squeeze()
-        sample_array = sample_data.bands.data
-        sample_id = str(sample_data.sample.data)
-
+        chip_paths = self.chip_list[index]
+        label = self._get_image_array(chip_paths['LABEL'], squeeze=True)
```
Comment (williamh890) [inline severe-weather/loader.py]: I would lean towards doing `self._get_image_array(chip_paths['LABEL']).squeeze()` instead of adding a boolean `squeeze` param to `_get_image_array`

Comment (forrestfwilliams) [reply severe-weather/loader.py]: Yep, I agree with this.

**Review by williamh890 — APPROVED** [review]
